File: logica-programacao/javascript/026.js
/* Peça a idade de uma pessoa.
Mostre True se ela for adolescente (entre 13 e 17 anos inclusive), caso contrário False.
Use apenas operadores relacionais + lógicos. */

let idade = prompt("Idade: ");
let intIdade = parseInt(idade);

let adolescente = intIdade >= 13 && intIdade <= 17;

console.log(`Adolescente? ${adolescente}`);

File: logica-programacao/python/026.py
# Peça a idade de uma pessoa.
# Mostre True se ela for adolescente (entre 13 e 17 anos inclusive), caso contrário False.
# Use apenas operadores relacionais + lógicos.

idade = input("Idade: ")
int_idade = int(idade)

adolescente = int_idade >= 13 and int_idade <= 17

print(f"É adolescente? {adolescente}")